web/themes/joinup/templates/parts/header.html.twig

```diff
@@ -1,21 +1,23 @@
 <div class="container-fluid">
   <div class="region region--header mdl-layout__header-row">
-    <a href="/" class="header-logo mdl-layout-title" title="Home"></a>
+    <a href="/" class="header-logo mdl-layout-title" title="Home">
+      <img src="/themes/joinup/images/logo.png">
```

Comment:
Switch to {{ base_path ~ directory }} like it's done in the featured anonymous template.